apps/gameserver/include/Loot.h

```diff
@@ -87,12 +87,14 @@ class Loot : public Character
 public:
     static constexpr std::uint32_t DISAPPEAR_TIME = 180000;
     static constexpr std::uint32_t PRIORITY_TIME = 120000;
+    static constexpr std::int32_t MAX_DISTANCE_FROM_TARGET = 20;
```

Comment:
```suggestion
    static constexpr std::int32_t MAX_DISTANCE_FROM_THROWER = 20;
```
This is no to mistake from appear sight distance